content/basics/ably.textile

```diff
@@ -0,0 +1,86 @@
+---
+title: What is Ably?
+meta_description: "Ably is a realtime platform delivering powerful digital realtime experiences through a publish/subscribe network."
+---
+
+Ably’s realtime platform provides a scalable, fault-tolerant and worldwide publish/subscribe(pub/sub) network. Providing SDKs in a variety of programming languages with both REST and realtime interfaces Ably helps you get connected.
+
+This section will provide a short introduction to some of the key features and APIs of Ably, giving you an idea of what you can expect to build with Ably and how Ably's infrastructure helps you succeed.
+
+h2(#what). What can you build with Ably?
+
+Ably enables the distribution of realtime data to multiple users using the pub/sub messaging pattern with WebSockets, enabling you to build realtime experiences for your applications without having to build your own realtime infrastructure. Common use cares are:
+
+* Live Chat - chat for customer support or community engagement
+* Data Broadcast - broadcast realtime event data to millions of devices
+* Data Sync - keep backend and frontend clients in realtime sync
+* Multiplayer Collaboration - collaborative workspaces for synchronous teamwork
+* Notifications - deliver in-app notifications in realtime to users
+
+h2(#about). Ably features
+
+Ably provides a number of features that enhance realtime experiences, some of the key Ably features are: 
+
+* "Pub/sub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). Pub/sub with channels
+
+Ably uses the pub/sub architectural pattern to enable any number of publishers to publish data and any number of subscribers to subscribe to that data. This works with Ably's "concept of channels":/channels, enabling publishers to publish to a single location and subscribers to subscribe to a single source.
+
+h3(#presence-and-occupancy). Presence and occupancy
+
+Ably’s "presence":/presence-occupancy/presence feature enables clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events, such as a client entering or leaving the presence set. Each member in the presence set can have data associated with them. You can also request a list of clients or devices that are connected to a channel at a particular point in time via the REST API.
```

Comment:
Again, an example here could be "Presence could be used to power a live avatar stack to show what users are present in a virtual space (e.g. whiteboard, document, or worksheet)"